networkx/algorithms/tests/test_simple_paths.py

```diff
@@ -85,32 +85,32 @@ def test_multidigraph(self):
 def test_all_simple_paths():
     G = nx.path_graph(4)
     paths = nx.all_simple_paths(G,0,3)
-    assert_equal(list(list(p) for p in paths),[[0,1,2,3]])
```

Comment:
I use tuples for paths, so the order is preserved. The set is for the set of paths.
There is indeed a single path in this particular case, but I wanted it to be consistant with the ones below.